presidio-image-redactor/presidio_image_redactor/image_redactor_engine.py

```diff
@@ -42,9 +49,12 @@ def redact(
 
         image = ImageChops.duplicate(image)
 
-        bboxes = self.image_analyzer_engine.analyze(
-            image, ocr_kwargs, **text_analyzer_kwargs
-        )
+        if isinstance(self.image_analyzer_engine, QRImageAnalyzerEngine):
```

Comment:
@omri374 @vpvpvpvp Any idea on making it more open-close? 
Maybe a single ImageAnalyzerEngine that we inherit from with optional **ocr_kwargs?

Reply:
In case of direct inheritance of `QRImageAnalyzerEngine` from `ImageAnalyzerEngine`, it would only need to add ocr_kwars to the analyze method of `QRImageAnalyzerEngine`. This is probably the easiest way.

Potentially, it seems like the most optimal implementation when `ImageAnalyzerEngine` is used for orchestrating different recognizers (ocr recognizer, QR recognizer, etc.). In the vein of what was suggested earlier https://github.com/microsoft/presidio/pull/1036#issuecomment-1437483048.